### DISCUSSION TITLE
Does Ginan have PPP-RTK capabilities

### DISCUSSION THREAD

**Discussion by GNSSer123** [GeoscienceAustralia/ginan#81] [Q&A] [answered]
Dear Ginan team，
   Does Ginan have PPP-RTK capabilities？
   Thanks

**Comment by ronaldmaj** [comment, marked as answer]
Hi [GNSSer123](https://github.com/GNSSer123),

Thanks for your question, apologies for the delay in response. As a direct answer, Ginan does have PPP-RTK capabilities and setting up the right fields within our YAML configuration will allow you to run it in real-time with the relevant corrections coming in. 
However, we recently moved to version 3 of the Ginan software and are still working through updating our various example configuration files. This will provide a configuration file with those relevant fields all filled in.
Unfortunately, the PPP-RTK one is not ready for public release yet, but we are working on it.
Once we have that ready, that should be available on our develop-weekly branch. 
I will let you know when that has occurred.

In the meantime you can use `pea -Y 5` on the command line to produce a couple of useful files in your pwd. These are:
- `defaultConfiguration.md` which provides some useful documentation on configuration fields
- and `GinanYamlInspector.html` which you can run in your browser to explore the various fields via a friendlier interactive format

Please let us know if you have any other further questions or queries.
